README.md

```diff
@@ -36,7 +37,13 @@
 
 [](#__START_EMBEDDED_README__)
 
-
+## Docs
```

Comment:
Maybe this should be called "Additional docs", since we already have a "Documentation" link above linking to the Dart API docs?

Or maybe we don't rename it could just include a link in this section as well?

Reply:
Additional docs sounds good to me!